build/conda/conda-3-9-env-full.yaml

```diff
@@ -6,4 +6,3 @@ dependencies:
   - pip
   - poetry=1.1.13
   - lightgbm=3.3.3
-  - charset-normalizer=2.1.1
```

Comment:
Im not sure why this is in here.

Reply:
there were weird packaging errors related to the versioning of this package. let's try removing the pin and see if that works out

Reply:
So if you look at the versioning, I added this in the excel pr.  Was accidental and an issue with my rebase.

Hence why I removed it.

Reply:
🚀